docs/docs/developers/charts.md

```diff
@@ -103,10 +103,11 @@ class Custom extends Chart.controllers.bubble {
         ctx.restore();
     }
 });
+Custom.id = 'derivedBubble';
+Custom.defaults = Chart.defaults.bubble;
```

Comment:
I think you might be able to delete line 85 (`Chart.defaults.derivedBubble = Chart.defaults.bubble`) as a duplicate of this line?

Reply:
Yeah, removed bunch of outdated comments too.